refresh/index.js

```diff
@@ -300,23 +299,28 @@ module.exports = Generator.extend({
       this.itemsToIgnore = []
 
       // Package dependencies to add to Package.swift
-      // eg this.dependencies.push('.Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 1, minor: 7),')
+      // eg this.dependencies.push('.package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextMinor(from : "1.7.0")),')
       this.dependencies = []
 
+      // Module Dependencies to add to Package.swift
+      this.modules = []
+
       // Initialization code to add to Application.swift by code block
       // eg this.appInitCode.services.push('try initializeServiceCloudant()')
       this.appInitCode = {
         capabilities: [],
         services: [],
+        service_imports: [],
```

Comment:
These changes are from https://github.com/IBM-Swift/generator-swiftserver/pull/347 so I would like to delay reviewing the rest of this PR until that is merged.